tests/__init__.py

```diff
@@ -120,3 +122,41 @@ def skip_on_travis(test_item):
         return unittest.skip("Test doesn't run on travis.")(test_item)
     else:
         return test_item
+
+class CaptureStdout(list):
```

Comment:
This is a cool construction. However, please consider using existing [contextlib.redirect_stdout](https://docs.python.org/3/library/contextlib.html#contextlib.redirect_stdout) functionality from the contextlib library.

Reply:
Opened #2355 so we can address this when Python 2 support is dropped

Reply:
In general, it is preferable to keep our code as simple as possible to limit potential bugs and technical debt. In this case, the data-store can use `contextlib.redirect_stdout` without waiting for Python 2 support to be dropped from the  CLI.